back/spec/lib/participation_method/voting_spec.rb

```diff
@@ -3,41 +3,41 @@
 require 'rails_helper'
 
 RSpec.describe ParticipationMethod::Voting do
-  subject(:participation_method) { described_class.new project }
+  subject(:participation_method) { described_class.new context }
 
-  let(:project) { create(:continuous_budgeting_project) }
+  let(:context) { create(:continuous_budgeting_project) }
 
   describe '#assign_defaults_for_participation_context' do
     context 'budgeting' do
-      let(:project) { build(:continuous_budgeting_project) }
+      let(:context) { build(:continuous_budgeting_project) }
 
-      it 'sets the posting method to unlimited and ideas order to random' do
```

Comment:
add back in `and ideas order to random`

Reply:
Strange it disappeared, I tried to make sure it to take the version from i1 here. I'll bring it back.